app/components/dashboard/run/left-panel/template.hbs

```diff
@@ -129,4 +104,13 @@
             </div>
         </div>
     </div>
+
+     {{#if showDimmer}}
+    <div class="ui segment" style="position:fixed; top:25%; width:50%; left:25%; height:35%; z-index:999">
+      <div class="ui active dimmer">
+        <div class="ui indeterminate huge text loader">Zipping your Tale</div>
+    </div>
+    <p></p>
+  </div>
+  {{/if}}
```

Comment:
I didn't see the dimmer in your new GIF... if it is no longer going to display, we should probably just remove it